HumanControls/DashPage.cpp

```diff
@@ -4,9 +4,39 @@ DashPage::DashPage() : Page(Page::PageType::DASH_PAGE)
 {
 }
 
-void DashPage::paint(DisplayController &display, bool isActivated, const char *status)
+void DashPage::paint(DisplayController &display, bool isActivated, JsonState &state)
```

Comment:
Use a reference to JsonObject instead of JsonState. This means you don't have to call `.root()` over and over again.

Reply:
Done.